wrappers/interface/IGstWrapper.h

```diff
@@ -78,6 +78,12 @@ class IGstWrapper
      */
     virtual void gstInit(int *argc, char ***argv) = 0;
 
+    /**
+     * @brief Deinitalise gstreamer.
+     *
+     */
+    virtual void gstDeinit() = 0;
```

Comment:
dead code - you don't call this function

Reply:
Well, diff shows, that you added it :p Is gst_deinit additional call needed to fix any of valgrind issues?